extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -257,6 +262,42 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * If requestsFrameUrl (and optionally dataHash as well) are specified in
+   * config, check whether third-party iframe already exists, and if not,
+   * create it.
+   * @private
+   */
+  handleRequestsFrameUrl_() {
+    if (!this.config_['requestsFrameUrl']) {
+      return;
+    }
+    const requestsFrameKey = this.buildRequestsFrameUrlWithDataHash(
```

Comment:
So you mean if slot one has transport: { iframe: 'foo.html', dataHash: 'abc' } and slot two has transport: { iframe: 'foo.html', dataHash: 'xyz' } then I should create a single iframe? Can do, but this is different than my understanding of it so just double-checking to make sure.